### ISSUE TITLE
Feature: Disable auto post to my page

### ISSUE THREAD

**Issue by chrisbergr** [aaronpk/OwnYourSwarm#45]
I get that this is the core feature of OwnYourSwarm, but it would be great to have the possibility to deactivate the auto post. -Like all the other toggles on the dasboard.

I have the strange problem that the location data is not transferred in the auto post. With the manual import everything works smoothly.

I'm assuming it's a problem on my end, but until I figure that out, the feature is creating some chaos.

**Comment by aaronpk** [comment]
That's strange because they should be sending exactly the same data to your site in both cases. I can't imagine how the location data is missing from the one that posts automatically.

**Comment by chrisbergr** [comment]
Yes, as I said, I assume that this is caused by my website. It also happens that when a new auto post is made, the previous checkin is also reset and the imported images are gone. Super strange. I suspect it has something to do with one of the plugins for caching or image optimization, but it will take a while to find out.

This is just my specific case where it would be helpful for me. But maybe there is also a general benefit to be able to disable this optionally for others, too.

**Comment by aaronpk** [comment]
The easiest way to disable imports is to click the "disconnect foursquare" button on the dashboard. That will stop the auto-imports until you reconnect your foursquare account.
